dragonboard/tools/create_fake_data.py

```diff
@@ -0,0 +1,106 @@
+'''
+Create dragonboard data files with random noise
+
+Usage:
+    create_fake_pedestals.py <outputfile> [options]
+
+Options:
+    -v <v>, --version=<v>      file version for the output file,
+                               v5_1_05 or v5_1_0B [default: v5_1_0B]
+
+    -n <N>, --num-events=<N>   Number of events to produce [default: 100]
+'''
+import numpy as np
+import struct
+from docopt import docopt
+
+
+def write_header_v5_1_0B(
+        f,
+        pps_counter,
+        event_counter,
+        trigger_counter,
+        counter_10MHz,
+        counter_133MHz,
+        ):
+    header_bytes = struct.pack(
+        '!HHIIIQQ16s',
+        0xaaaa,
+        pps_counter,
+        counter_10MHz,
+        event_counter,
+        trigger_counter,
+        counter_133MHz,
+        0xdddddddddddddddd,
+        b'\xf0\x01\xf0\x03\xf0\x01\xf0\x00\xf0\x00\xf0\x00\xf0\x03\xf0\x01'
```

Comment:
The assumption, that all of these flags are always the same is not true:
~~~
In [7]: set_of_flags = set(e.header.flag for e in db.read("Ped_Rand10kHz_No1_IP90.dat"))

In [8]: len(set_of_flags)
Out[8]: 33
~~~

Just mentioning it ... the writer is a good thing .. and this is no reason no to merge.